src/App.tsx

```diff
@@ -88,6 +88,7 @@ const App = () => {
   const isFiltersPresetModalVisible = useAppSelector(state => state.ui.filtersPresetModal?.isOpen);
   const isInClusterMode = useAppSelector(isInClusterModeSelector);
   const isNewResourceWizardVisible = useAppSelector(state => state.ui.newResourceWizard.isOpen);
+  const isKubeConfigBrowseSettingOpen = useAppSelector(state => state.ui.kubeConfigBrowseSetting.isOpen);
```

Comment:
Lets name it with `Settings` instead of `Setting` and change accordingly all over the project

Reply:
Okay